2017-11-17 06:17:18,525 - streamparse.stdout - INFO - Could not create tcount
2017-11-17 06:17:18,549 - streamparse.stdout - INFO - could not create table
2017-11-17 06:17:18,550 - streamparse.stdout - INFO - ucount = 
2017-11-17 06:17:18,551 - streamparse.stdout - INFO - 1
2017-11-17 06:17:18,551 - streamparse.stdout - INFO - uword = 
2017-11-17 06:17:18,551 - streamparse.stdout - INFO - are
2017-11-17 06:17:18,554 - streamparse.storm.component - ERROR - Exception in WordCounter.run()
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/streamparse-2.1.4-py2.7.egg/streamparse/storm/component.py", line 489, in run
    self._run()
  File "/usr/lib/python2.7/site-packages/streamparse-2.1.4-py2.7.egg/streamparse/storm/bolt.py", line 219, in _run
    self.process(tup)
  File "bolts/wordcount.py", line 65, in process
    cur.execute("UPDATE tweetwordcount SET count=%s WHERE word=%s", (uCount, uWord))
InternalError: current transaction is aborted, commands ignored until end of transaction block

